lib/vendor/PEAR/Command/Channels.php

```diff
@@ -164,7 +164,7 @@ function doList($command, $options, $params)
     {
         $reg = &$this->config->getRegistry();
         $registered = $reg->getChannels();
-        usort($registered, array(&$this, '_sortchannels'));
+        usort($registered, array($this, '_sortchannels'));
         $i = $j = 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$i" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$j" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)